packages/postgraphile-core/__tests__/kitchen-sink-data.sql

```diff
@@ -84,8 +84,8 @@ insert into b.types values (
   '2016-10-07',
   '04:05:06',
   '04:05:06 -8:00',
-  '1 year 2 months 3 days 4 hours 5 minutes 6 seconds',
-  ARRAY['1 year 2 months 3 days 4 hours 5 minutes 6 seconds', '1 year 1 months 1 days 1 hours 1 minutes 1 seconds']::interval[],
+  '1 year 2 months 3 days 4 hours 5 minutes 6.5 seconds',
+  ARRAY['1 year 2 months 3 days 4 hours 5 minutes 6 seconds', '1 year 1 months 1 days 1 hours 1 minutes 1.234 seconds', '123.456 milliseconds']::interval[],
```

Comment:
Use more troublesome values here; PostgreSQL supports nanosecond precision:

```suggestion
  '1 year 2 months 3 days 4 hours 5 minutes 59.876543210 seconds',
  ARRAY['1 year 2 months 3 days 4 hours 5 minutes 6 seconds', '1 year 1 months 1 days 1 hours 1 minutes 1.234567890 seconds', '123.456789012 milliseconds']::interval[],
```